<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Channel_simClass</class>
 <widget class="QMainWindow" name="Channel_simClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>模拟无线多体制发送端</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Channel_sim.qrc">
    <normaloff>:/Channel_sim/resouce/avic.png</normaloff>:/Channel_sim/resouce/avic.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <property name="icon">
     <iconset resource="Channel_sim.qrc">
      <normaloff>:/Channel_sim/resouce/file.png</normaloff>:/Channel_sim/resouce/file.png</iconset>
    </property>
    <addaction name="file_action"/>
   </widget>
   <widget class="QMenu" name="view_action">
    <property name="title">
     <string>视图</string>
    </property>
    <property name="icon">
     <iconset resource="Channel_sim.qrc">
      <normaloff>:/Channel_sim/resouce/列表视图.png</normaloff>:/Channel_sim/resouce/列表视图.png</iconset>
    </property>
    <addaction name="actionchannel"/>
    <addaction name="actionlog"/>
    <addaction name="actionvideo"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="view_action"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="minimumSize">
    <size>
     <width>136</width>
     <height>274</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>139</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>121</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="Channel1_checkbox">
        <property name="text">
         <string>信道1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="packetlost_bar1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lostrate_label1">
        <property name="text">
         <string>丢包率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="channel2_checkbox">
        <property name="text">
         <string>信道2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="packetlost_bar2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lostrate_label2">
        <property name="text">
         <string>丢包率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="channel3_checkbox">
        <property name="text">
         <string>信道3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="packetlost_bar3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lostrate_label3">
        <property name="text">
         <string>丢包率：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="tooldock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>57</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>57</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>64</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Play_btn">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {
    /* 鼠标按下状态 */
    background-color: #999;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Channel_sim.qrc">
          <normaloff>:/Channel_sim/resouce/开始1.png</normaloff>:/Channel_sim/resouce/开始1.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Stop_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {
    /* 鼠标按下状态 */
    background-color: #999;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Channel_sim.qrc">
          <normaloff>:/Channel_sim/resouce/24gf-stop.png</normaloff>:/Channel_sim/resouce/24gf-stop.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="file_action">
   <property name="icon">
    <iconset resource="Channel_sim.qrc">
     <normaloff>:/Channel_sim/resouce/24gf-folderOpen-copy.png</normaloff>:/Channel_sim/resouce/24gf-folderOpen-copy.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionchannel">
   <property name="text">
    <string>channel</string>
   </property>
  </action>
  <action name="actionlog">
   <property name="text">
    <string>tool</string>
   </property>
  </action>
  <action name="actionvideo">
   <property name="text">
    <string>video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Channel_sim.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionchannel</sender>
   <signal>triggered()</signal>
   <receiver>dockWidget_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionlog</sender>
   <signal>triggered()</signal>
   <receiver>tooldock</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
